resource/doc/ffmpeg.md

```diff
@@ -12,6 +12,7 @@ make install
 Using FFmpeg+libvmaf is very powerful, as you can create complex filters to calculate VMAF directly on videos of different encoding formats and resolutions. For the best practices of computing VMAF at the right resolution, refer to our [techblog](https://medium.com/netflix-techblog/vmaf-the-journey-continues-44b51ee9ed12). 
 
 We provide a few examples how you can construct the FFmpeg command line and use VMAF as a filter. Note that you may need to download the test videos from [vmaf_resource](https://github.com/Netflix/vmaf_resource/tree/master/python/test/resource).
+If all of this is too confusing and you've already installed libvmaf (easiest through homebrew or any package manager) and configured ffmpeg you could use [this](https://gist.github.com/Audition-CSBlock/bef34e553132efad883c0f128c46d638) shell script.
```

Comment:
Would you like to add this script to the repo instead of providing an external link? This way it is easier to maintain.